### PR TITLE
Code-style and fixed on .eslintrc.js

### PR DESCRIPTION
**Pull request by Yash-Singh1**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,8 +5,8 @@
   "scripts": {
     "api": "node tasks/build.js --api",
     "benchmark-server": "node tests/benchmark-server/index.js",
+    "code-style": "eslint --fix \"src/**/*.ts\" \"src/**/*.tsx\" \"tasks/**/*.js\" \"tests/**/*.ts\" \"tests/**/*.js\" \"index.d.ts\" \".eslintrc.js\"",
```
Comment (Gusted) [inline package.json:8]: Preserve alphabetic order in the scripts section.

**Review by Gusted — CHANGES_REQUESTED** [review]

**Comment by Yash-Singh1** [comment]
@Gusted Should `.eslintrc.js` be linted in the `lint` script?

File: .eslintrc.js [hunk 2]
```diff
@@ -32,6 +33,7 @@ module.exports = {
         '@typescript-eslint/array-type': ['error', {
             default: 'array-simple',
         }],
+        'yoda': ['error', 'never'],
```
Comment (alexanderby) [inline .eslintrc.js:36]: Wait what, Yoda?
image

**Review by alexanderby — APPROVED** [review]

File: .eslintrc.js (hunk 2 above)
Comment (Gusted) [reply .eslintrc.js:36]: Yeah it's apparently for this line https://github.com/darkreader/darkreader/pull/4423/files#diff-aab4ceb6d3254a7b948f95d8057cd8d91cd43c09f95569885cbce6d8b079d926R154

**Pull request merged**